src/ocean/text/json/Json.d

```diff
@@ -31,60 +31,64 @@ import ocean.text.json.JsonParser;
 
 import Float = ocean.text.convert.Float;
 
-version(UnitTest) import ocean.core.Test;
+version (UnitTest)
+{
+    import ocean.core.Test;
+    import ocean.io.Stdout;
+}
+
+///
+unittest
+{
+    void example ( )
+    {
+        /*
+            Typical usage is as follows:
+        */
+        auto json = new Json!(char);
+        json.parse(`{"t": true, "n":null, "array":["world", [4, 5]]}`);
+
+        /*
+            Convert back to text format:
+        */
+        Stdout.formatln("{}", json.value.print());
```

Comment:
Just want to check we need this output or can we use `test`?

Reply:
I guess it's a matter of taste, since the `example()`s are not run as unit tests any way. I've put in the `test` statements to show without running what the expected output should be and the `Stdout.formatln` statements to make it easy to adjust things and view the outputs in one place on the console when pasting the `example()` contents into a standalone main module for exploration.